core/client/fs/src/test/java/alluxio/client/lineage/LineageFileOutStreamTest.java

```diff
@@ -21,7 +21,7 @@
 import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mockito;
+static import org.mockito.Mockito;
```

Comment:
See https://github.com/Alluxio/alluxio/pull/6725/files for an example of how to use static imports